### ISSUE TITLE
Another app called seal on the play store

### ISSUE THREAD

**Issue by Brandonbr1** [JunkFood02/Seal#1160]
### Checklist

- [ ] I'm reporting a bug unrelated to a specific site.
- [X] I've verified that I'm running the [**latest version**](https://github.com/yt-dlp/yt-dlp/releases/latest) of yt-dlp.
- [X] I've verified that I'm running the latest [**stable version**](https://github.com/JunkFood02/Seal/releases/latest/) of Seal or any later [**preview versions**](https://github.com/JunkFood02/Seal/releases).
- [X] I've read the [**Contributing guidelines**](https://github.com/JunkFood02/Seal/blob/main/CONTRIBUTING.md) and [**Code Of Conduct.**](https://github.com/JunkFood02/Seal/blob/main/CODE_OF_CONDUCT.md)
- [X] I've checked that the site i'm trying to download from is in the [**Supported Sites**](https://github.com/yt-dlp/yt-dlp/blob/master/supportedsites.md) list from yt-dlp
- [X] I understand that the issue will be (ignored/closed) if I intentionally remove or skip any mandatory field.

### Describe the bug

not a bug but there is an app called seal downloadee by Hit K apps. which has the same icon as your app, while adding on more tracking filling users who don't know about f-droid or github

### To Reproduce

blank

### Error reports

blank

**Comment by Sanzium** [comment]
Some screenshots and info

![Screenshot_20231104_145150_Google Play Store.png](https://github.com/JunkFood02/Seal/assets/73362024/09817332-caf6-4423-881d-6161f7e34e44)

![Screenshot_20231104_145155_Google Play Store.png](https://github.com/JunkFood02/Seal/assets/73362024/a91b2074-37e0-47d4-8f0b-d042eabc139d)

![Screenshot_20231104_145136_Google Play Store.png](https://github.com/JunkFood02/Seal/assets/73362024/a7b7dbb5-bda7-43eb-b4df-c7acd6e08adb)

![Screenshot_20231104_144823_Google Play Store.png](https://github.com/JunkFood02/Seal/assets/73362024/de7fd6f5-2171-4c3a-8479-bcc3e1479859)

![Screenshot_20231104_145116_MiXplorer.png](https://github.com/JunkFood02/Seal/assets/73362024/825bf575-604c-483e-bee1-99e6cf544ab6)

They even have a "privacy policy" https://hitkapps.blogspot.com/p/seal-downloader-privacy-policy.html

![Screenshot_20231104_144930_MiXplorer.png](https://github.com/JunkFood02/Seal/assets/73362024/992ea836-db6b-4582-8733-d72c219c77f1)

**Comment by vaimalaviya1233** [comment]
really sad to see someone else taking credit of something that's not their.

**Comment by Marco-9456** [comment]
> > This is Impossible because Google is not allowing apps that download content from YouTube or YouTube Music
> >
> > _Originally posted by @Marco-9456 in https://github.com/JunkFood02/Seal/issues/129#issuecomment-1219539204_
> 
> And you said it can't be done when an imposter app made it to the Play Store 🫤

It's just using the app name and icon, the code is not the same + it's broken and doesn't work.

**Comment by fillwithjoy1** [comment]
This is so sad, I'm going to review bomb it and report bomb it as well

**Comment by DesolateIntention** [comment]
The dev should sue this POS app. It's absolute crap and filled with ads.

**Comment by fillwithjoy1** [comment]
We should mass report this on the Play Store and tell them they have stole Seal from GitHub. Oh also review bomb them linking to the actual app.

Sent with Shortwave <https://www.shortwave.com?utm_medium=email&utm_content=signature&utm_source=am95c2luZ2guanMxQGdtYWlsLmNvbQ==>

**Comment by Brandonbr1** [comment]
The sad part is the play store app has 500k downloads, and someone even said this is 'better' than the one on f-droid

**Comment by JunkFood02** [comment]
YouTube downloader -> Google hates you + Nobody cares about an app which doesn't exist on Play Store

Nothing much to say about those ppl who are using the fake app on the Play Store. But they probably don't deserve such a good app so let them go 🤌

**Comment by fillwithjoy1** [comment]
truly a depression moment :(

**Comment by Brandonbr1** [comment]
> YouTube downloader -> Google hates you + Nobody cares about an app which doesn't exist on Play Store
> 
> Nothing much to say about those ppl who are using the fake app on the Play Store. But they probably don't deserve such a good app so let them go 🤌

Yeah most people who use phones don't know quite basic technology stuff, so I am not surprised the ripoff got more downloads, because it is on the more popular app store.

Plus, alot of people are told, 'don't download anything besides from the play store', and they believe it, they don't know any better sadly.

Just keep developing this app.

**Comment by Harishwarrior** [comment]
How to sue this guy?
